<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>381</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>100</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>char</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="trialsSpinBox">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>55</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="value">
      <number>5</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>70</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>msec</string>
     </property>
    </widget>
    <widget class="QSlider" name="lengthSlider">
     <property name="geometry">
      <rect>
       <x>99</x>
       <y>100</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimum">
      <number>5</number>
     </property>
     <property name="maximum">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Number of trials</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="exposureLCD">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>70</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value" stdset="0">
      <double>100.000000000000000</double>
     </property>
     <property name="intValue" stdset="0">
      <number>100</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Length</string>
     </property>
    </widget>
    <widget class="QSlider" name="exposureSlider">
     <property name="geometry">
      <rect>
       <x>99</x>
       <y>70</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimum">
      <number>100</number>
     </property>
     <property name="maximum">
      <number>3000</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Exposure</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lengthLCD">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>100</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="numDigits">
      <number>2</number>
     </property>
     <property name="intValue" stdset="0">
      <number>5</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>180</y>
      <width>381</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Summary</string>
    </property>
    <widget class="QWidget" name="scoreWidget" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>361</width>
       <height>151</height>
      </rect>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>161</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Characters correct</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>141</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Total chars correct</string>
      </property>
     </widget>
     <widget class="QLabel" name="correctCharFraction">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>30</y>
        <width>71</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>--</string>
      </property>
     </widget>
     <widget class="QLabel" name="cumulativeFraction">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>60</y>
        <width>71</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>--</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>271</width>
      <height>351</height>
     </rect>
    </property>
    <property name="title">
     <string>SpeedReader</string>
    </property>
    <widget class="QPushButton" name="startButton">
     <property name="geometry">
      <rect>
       <x>52</x>
       <y>120</y>
       <width>171</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Start the trials</string>
     </property>
    </widget>
    <widget class="QLabel" name="responseLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Response String</string>
     </property>
    </widget>
    <widget class="QLabel" name="displayLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Target String</string>
     </property>
    </widget>
    <widget class="QPushButton" name="nextButton">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>310</y>
       <width>93</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="targetString">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>67</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>Get ready to watch this space</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="responseString">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Type the target string here</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
